### PR TITLE
Student Nikita Bondars lesson 1

### PR DESCRIPTION
**Pull request by NickBRiga**
Created 1st homework

### CODE REVIEW

File: src/main/java/students/student_Nikita_Bondars/Lesson_1/Task_12.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main(final String[] args){
+
+        final int a = 13;
+        final int b = 12;
+
+        System.out.println("Результат:");
+        System.out.println(a * b);
+
+}
+}
```
Comment (antdolg1) [inline src/main/java/students/student_Nikita_Bondars/Lesson_1/Task_12.java:13]: Please format line 12 and 13

```
    }
}
```

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please try to use english when outputting text to console and adding comments. It will help you in future.

File: src/main/java/students/student_Nikita_Bondars/Lesson_1/Task_11.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package main.java.students.student_Nikita_Bondars.Lesson_1;
+
+public class Task_11 {
+    public static void main(final String[] args)  {
```
[the remark this answers is not in the document]
Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/Lesson_1/Task_11.java:4]: Fixed

File: src/main/java/students/student_Nikita_Bondars/LessonWork2/Variables.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.println("myInt value is: " + myInt);
+
+        long myLong = 1_000_000;
+        System.out.println("myLong value is: " + myLong);
+
+        float myFloat = 3.123456789F; // Либо маленькая буква f
+        System.out.println("myFloat value is: " + myFloat);
+
+        double myDouble = 3.123456789;
+        System.out.println("myDouble value is: " + myDouble );
```
[the remark this answers is not in the document]
Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/LessonWork2/Variables.java:22]: Fixed

**Review by antdolg1 — APPROVED** [review]
All good

**Pull request merged**